ansible/private-docker-registry/roles/docker-registry/templates/self-csr.json.j2

```diff
@@ -0,0 +1,19 @@
+{
+  "CN": "docker-registry",
+  "hosts": [
+    "{{ ansible_default_ipv4.address }}",
```

Comment:
if we plan to use this long-term, it would be nice to optionally set a FQDN here